### ISSUE TITLE
documentation request: object tracking

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/documentation-wiki#149]
https://github.com/ArctosDB/arctos-webinars/issues/33#issuecomment-618027250

We should do a better job of illuminating the object tracking situation, somewhere/somehow. It's one of the things that lead to Arctos being created and the container model being developed (after much experimenting with magnetic barcodes with the wrong numbers printed on them). I think using something like taxonomy (and/or geography, etc.) for object tracking is just "normal," everybody does it, nobody seems to think too much about it. Arctos offers two alternatives, both of which track the actual thing that needs tracked (parts) instead of the thing that got cataloged (an idea that was firmed up, if not developed, through the magnetic barcode experiments), neither of which are necessarily confounded with any unrelated data.

I wonder if anyone's ever looked at how much specimen degradation is ultimately attributable to things like shelf-shuffling after taxonomic changes vs. scientific usage?

**Comment by Jegelewicz** [comment]
This is definitely on my radar - see [my comment in the original issue](https://github.com/ArctosDB/arctos-webinars/issues/33#issuecomment-618521524).

**Comment by Jegelewicz** [comment]
> I wonder if anyone's ever looked at how much specimen degradation is ultimately attributable to things like shelf-shuffling after taxonomic changes vs. scientific usage?

A perfect SPNHC paper, but I'm sure no one has done before and after condition reports (nat hist people rarely do them at all). Just the exposure to light and pests tho....

**Comment by campmlc** [comment]
Yes, definitely need a paper on this. Also quantify staff time and storage
resources.
